guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -5,13 +5,23 @@
 If the custom code is executed before and/or after the provisioning process, use custom provisioning snippets to avoid recreating cloned templates.
 For more information about configuring custom provisioning snippets, see {ProvisioningDocURL}Creating_Custom_Provisioning_Snippets_provisioning[Creating Custom Provisioning Snippets] in _{ProvisioningDocTitle}_.
 
+* Optional: {Project} creates the `/var/lib/pgsql/data-old/` directory for backup purposes during the upgrade. If the upgrade fails or {Project} loses PostgreSQL data shortly after the upgrade, you can use this directory to restore your external PostgreSQL database.
++
+However, after several database updates, the backup becomes obsolete, and cannot be used for disaster recovery anymore. When this occurs, or if you do not want to keep the data after a successful update, delete the directory:
```

Comment:
When discussing this, we ended up that this step shouldn't be optional.

Especially because we realized that the data becomes instantly unusable (for a restore) once the PostreSQL 12-to-13 upgrade is finished. So I'd just say "it's a backup, please remove once the upgrade is complete"

Reply:
Ah, I see, then my initial draft was actually correct in this regard. :-) Will rewrite accordingly.

Thank you!